src/models/movie.ts

```diff
@@ -1,39 +1,32 @@
 export interface Movie {
   id: string;
-  name: string;
-  urlName: string;
+  title: string;
+  url: string;
   releaseDate: string;
-  description: string;
+  overview: string;
   rating: number;
-  genre: { value: string, label: string }[];
+  genre: { value: string; label: string }[] | [];
   runtime: number;
 }
 
-export interface State {
-  name: string;
+export interface formInitialForMovie {
```

Comment:
```suggestion
export interface FormInitialForMovie {
```
But overall it's the name is misleading. It simply describes the form structure, regardless of whether it's an empty value or an actual movie. Maybe `MovieFormProps` or something like that would be better.